packages/slate-editor/src/extensions/placeholders/elements/CoveragePlaceholderElement.module.scss

```diff
@@ -0,0 +1,9 @@
+@import "styles/variables";
+
+.Action {
```

Comment:
Why uppercase? 🤔

Reply:
I was just following the casing that's already in other modules, but it doesn't make too much sense to me so I'll change it to camelCase.